include/mbedtls/build_info.h

```diff
@@ -87,6 +87,18 @@
 #define MBEDTLS_MD_C
 #endif
 
+/* PSA crypto specific configuration options
+ * - If config_psa.h reads a configuration option in preprocessor directive,
```

Comment:
I would like to have more precise rules, that don't require the reader to read all of `config_psa.h` (especially because it's pretty hard to read). Where `build_info.h` has a rule `#if A #define B`, I want to know, based on A and B, whether this rule can be put before the inclusion of `config_psa.h`, after, or neither. And I'd also like . But we improve in a follow-up.